src/ContainerFactory.ts

```diff
@@ -17,13 +20,26 @@ export function createContainer(): Container {
         password: getEnvOrDefault('CACHE_PASSWORD', ''),
         db: Number(getEnvOrDefault('CACHE_DB'))
     }));
-    const sessionHandler = SessionMiddleware(config, sessionStore);
+    container.bind(SessionStore).toConstantValue(sessionStore);
+    container.bind(SessionMiddleware).toConstantValue(SessionMiddleware(config, sessionStore));
+    container.bind(AuthMiddleware).toConstantValue(new AuthMiddleware());
 
-    const authMiddleware = new AuthMiddleware();
+    const kafkaClient = new kafka.KafkaClient({
+        kafkaHost: getEnvOrDefault('KAFKA_BROKER_ADDR'),
+        requestTimeout: 4000 // 4 seconds
```

Comment:
should this be configurable? why 4 seconds?

Reply:
4s seemed sensible to me, original 10s seemed bit too much. I will make it configurable.

Reply:
On top of that, the reason I am trying to keep it short is because transition waits for an email to be put into email sending queue. I do not want to wait too long to trigger timeouts on proxy side.

Reply:
Ok, makes sense - thanks for making this configurable